cyber-resilience-sig/whitepapers/stewards-and-cra.md

```diff
@@ -76,18 +76,22 @@ In Open Source Projects, Security Policies are public (then the publication requ
 the Project site and/or included in their source code repository.
 
 In practice, we see two patterns related to security policies:
-- Either the supporting organization publishes one, and Projects link to it in their documentation. This may include some minor
-clarifications, for example a specific link or email address to report vulnerabilities in that specific Project.
-- Or, Projects publish their policies based on the template provided by the supporting organization.
+- The supporting organization (Steward) creates and publishes one, and Projects agree to that policy, and link to it in their documentation.
+Project may make minor clarifications, for example add a specific link or email address to report vulnerabilities in that specific Project.
+- The supporting organization provides a template, and Projects publish their policies based on that template. In this case the number of
+changes in the Policy may be more significant.
+- The Project has a long-established public policy and that policy gets adapted by the Steward as their policy in the CRA terms.
 
-In both cases, both the Steward and the Project will need to check if the Policy they provide includes all needed elements and that it
-is followed. In the case the Project does not follow the Policy, the Steward needs to define the way to follow, see the [related FAQ entry](https://github.com/orcwg/cra-hub/issues/292).
+In all cases, both the Steward and the Project will need to check if the Policy they provide includes all needed elements and that it
+is followed. In the case the Project does not follow the Policy, the Steward needs to define the action plan, see the [related FAQ entry](https://github.com/orcwg/cra-hub/issues/292).
 
 As the Policy needs to be followed, the Steward and the Project will need to agree on a way to create the policy and provide changes.
```

Comment:
Does "changes" in this context mean filling in the blanks in a standard template or is it referring to the ongoing maintenance of the policy? Happy to suggest alternative wording once I know the intended meaning.